src/diverse_seq/records.py

```diff
@@ -608,7 +613,7 @@ def apply_app(
                 show_progress=True,
             ):
                 if not r:
-                    dvs_util.print_colour(r, style="red")
+                    dvs_util.print_colour(r, "red")
```

Comment:
this change unnecessary

Reply:
I believe at some point the `style` keword was changed to `colour` (I encountered an error here while trying to set things up and just changed it to the style used elsewhere)